src/__test__/App.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable*/
+describe ('Test ounters', () => {
+
+  it('Comments counters', () => {
+    expect(5).toBe(5);
+  })
+})
+
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
+
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
```

Comment:
- [ ] This test is not needed in the repo.  Either add some tests or please remove this file (and the folder if not using it). 👍